test/integration/driver.go

```diff
@@ -236,14 +240,15 @@ func makeRequests(pods map[string]string) map[string][]string {
 	return out
 }
 
+// checkAccessLogs searches for request ids in the access logs
 func checkAccessLogs(pods map[string]string, ids map[string][]string) {
 	n := 0
```

Comment:
Similar comment to line 207 re: pulling in 'n' initialization and increment into the for clause.